docs/installation/installing-tentacles/polling-tentacles-web-sockets.md

```diff
@@ -0,0 +1,91 @@
+
+!toc
+
+## Server Setup
+Octopus can share HTTPS ports with any application that uses the standard Windows networking library (HTTPSYS). This includes IIS, .NET apps and Octopus itself. It does not include any applications that utilise non-HTTPSYS TCP/IP or HTTP stacks. Check your product's documentation for details. SSL offloading and sharing HTTP ports is not supported.
+
+### Listen Address
+
+The first step is to select a URL listen prefix. HTTPSYS handles the initial SSL handshake and then routes the request based on the HTTP headers. This means that the request can be routed based on IP, hostname and path. See the [UrlPrefix documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/aa364698(v=vs.85).aspx) for available 
+syntax and how routes are matches. 
+
+In most cases, we recommend using `+` as the host name and a unique string for path. This will ensure that address
+takes the highest precedence. For example, to listen on port 443: `https://+:443/OctopusComms`. The path should not be 
+used by the other applications listening on the port.
+
+Once selected the Octopus Server can be configured to listen on that prefix using the following commands:
+```
+.\Octopus.Server.exe service --instance OctopusServer --stop
+.\Octopus.Server.exe configure --instance OctopusServer --commsListenWebSocket https://+:443/OctopusComms
+.\Octopus.Server.exe service --instance OctopusServer --start
+```
+
+### Testing
+To confirm that the server is successfully configured, open the listen address in your browser. If you are using `+` for the host, replace that with `localhost`. For example `https://localhost:443/OctopusComms`.
+
+If you encounter a certificate warning, ignore it and continue. You should get a page titled `Octopus Server configured successfully`. 
+
+If you do not yet know the thumbprint of your SSL certificate, you can find it by using your browser's developer tools.
+
+If you get your other application or an error, check the [server log file](/reference/log-files.md) to ensure the server is listening on the
+expected prefix (look for `Listener started`)
+
+## Tentacle Setup
+The setup of a WebSocket tentacle is the same as a TCP polling tentacle, except for the thumbprint and the command line option to specify the communications address.
+
+### Registering
+When issuing the `register-with` command during tentacle setup, omit the `--server-comms-port` parameter and specify the `--server-web-socket <address>` parameter. The address to use is the listen prefix (replacing `+` with the hostname) and `https` replaced with `wss` (e.g. `wss://MyServer:443/OctopusComms`). For example:
```

Comment:
instead of "MyServer", use "octopus.example.com" - the example.com/example.net/example.org domain names are there for this purpose.